l10n_br_fiscal/views/document_line_view.xml

```diff
@@ -73,15 +74,36 @@
           <page name="amounts" string="Amounts">
             <group>
               <group>
-                <field name="amount_untaxed" />
-                <field name="amount_fiscal" />
-                <field name="amount_tax" />
-                <field name="amount_estimate_tax" />
+                <field
```

Comment:
Pelo o que vi apenas os campos amount_estimate_tax e amount_tax_withholding permitem alteração na tela, portanto não vejo necessidade de incluir o atributo reandonly nos outros. E uma outra questão, mesmo esses dois campos não deveriam também ser do tipo compute e consequentemente não permitirem a edição direta deles? 
![image](https://user-images.githubusercontent.com/6341149/126212511-5cc68c27-35ee-4920-8c87-882e38e7856d.png)

Reply:
Entra no mesmo problema do comentário anterior: como está no modo manual, eu pensei que seria papel do usuário inserir todas as informações, inclusive essa dos montantes. 

Confesso que fiquei um pouco em dúvida no momento em que fiz essa alteração, e seria maneiro se pudesse ser discutido isso.

Reply:
Os campos do tipo compute já são readonly automaticamente, os únicos valores de totais que permitem alteração são esses dois, por isso não há sentido em incluir o atributo readonly nesses campos. Esses Valores Totais não podem ser diferentes do calculado se houver essa necessidade mesmo nos casos de Manual ou Semi-automatico é preciso alterar as informações que geram o Total porque a SEFAZ valida essa diferença. A questão se esses dois campos devem ser também calculados é independente desse PR, pelo o que entendo sim, pela razão de que acredito que a SEFAZ também valida esses campos, pelo código que existe hoje parece que em todos os casos, por enquanto, deve existir a possibilidade de alteração desses dois campos.

Reply:
@mbcosta eu chequei todos os 19 campos de taxas que tem um `'rel_tax_calc', '=', 'tax_calc_manual'` na visão e esses sao apenas campos m2o para registros de aliquotas fiscal.tax. Eu chequei os mappings da nfe no l10n_br_nfe e nenhum mapping de nfe usa esses esses m2o. O campos de m2o que sao exportados na NFe/SPED são os campos de *_cst_id por examplo, mas que não são usados na NFe. 

**Agora sim, o @mbcosta tem razão**, hoje no l10n_br_fiscal, se vc edita o campo ipi_tax_id tem um onchange que rola e que vai assignar o campo ipi_cst_id.

**Não teria utilidade vc poder ajustar uma aliquota de IPI se vc nao puder ajustar ipi_cst_id** como esta sendo feito aqui aqui por examplo:
https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py#L574

Porem se vc for mexer mais nisso, por favor toma cuidado em manter algo simples porque a gente tem que ver o custo/beneficio de uma funcionalidade dessa se ela for entrar no modulo l10n_br_fiscal que ja é muito grande...

cc @renatonlima

Reply:
Entendo. No caso seria deixar da forma como estava?

Reply:
@mbcosta batendo o olho no codigo eu tb pensei que campos como o ipi_cst_id que são usados no mapping da NFe  (para pegar apenas o examplo do IPI) nao estariam editaveis e que isso seria um problema. Mas eles estão sim felizmente. Eu estou continuando os testes...